docs/intro.rst

```diff
@@ -21,9 +21,12 @@ Main features
 - Reading routines for many ground based observation databases, such as:
 
 	- `AERONET <https://aeronet.gsfc.nasa.gov/>`_ Sun, SDA and Inversion products.
-	- `EBAS database <http://ebas.nilu.no/>`__.
-	- `GHOST` dataset (see e.g., `Petetin et al., 2020 <https://acp.copernicus.org/articles/20/11119/2020/acp-20-11119-2020.html>`_ for more information).
 	- `EARLINET Lidar network <https://www.earlinet.org/index.php?id=earlinet_homepage>`__.
+	- `EBAS database <http://ebas.nilu.no/>`__.
+	- `EEA AqEREP files <https://www.eea.europa.eu/data-and-maps/data/aqereporting-9>`__.
```

Comment:
[This page](https://www.eea.europa.eu/data-and-maps/data/aqereporting-8) suggest using the following name:
Air Quality e-Reporting (AQ e-Reporting)